### DISCUSSION TITLE
Welcome to openrport Discussions!

### DISCUSSION THREAD

**Discussion by alhdo** [openrport/openrport#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by buzzzo** [comment]
Hi

I'm a rport oss user and also just today i've learned that rport oss in dead, then just land on this repo. 
First question is: is it possible to connect my existing install to you repo in the sense that future updates could be managed from here ? (eg: update script, paring site etc etc)

Thx

**Comment by alhdo** [comment]
Hello, 

I am working on bring it back. I will use the same component as before like the update script will be likely the same. So when it get back online the update script will update your existing installation. The pairing service is already online. It needs some fix to point to the correct binary. I will give instruction in the wiki on how to switch the open version of the pairing service.
         
**The recap is yes you will be able to update from this repository.** 

Thanks

**Reply by buzzzo** [reply]
Thx for clarification.

Do you plan to add any bug fix / enhancement to the code or just keep it open ?

Thx

**Reply by alhdo** [reply]
I will maintain the project and fix the bug enhance it. The goal is to keep an opensource version alive an add new functionalities. If you find some bugs please open an issue. If you would like to collaborate please open a PR.

I create a discord server for the community to guide the direction of the development if you want to join. [Discord](https://discord.gg/HQ4wMQmzcu)

Thank you.

**Comment by 3gmgithub** [comment]
So happy you took this task on! I was wondering if anyone knows how to extend the functionality to use SSO, like the old RPort Plus plugin?
